test/any.specs.ts

```diff
@@ -0,0 +1,158 @@
+import "mocha"
+import fc, { Value } from "fast-check"
+import { areEquals, isEmpty, isNotEmpty } from "../src/any"
+
+//isEmpty tests
+describe("isEmpty: ", () => {
+  it("Should return true for empty strings", () =>
+    fc.assert(
+      fc.property(fc.string({ maxLength: 0 }), (value) =>
+        isEmpty(value)
+      )
+    ))
+
+  it("Should return false for non-empty strings", () =>
+    fc.assert(
+      fc.property(fc.string({ minLength: 1 }), (value) => !isEmpty(value)
+      )
+    ))
+
+  it("Should return true for empty arrays", () =>
+    fc.assert(
+      fc.property(fc.array(fc.nat(), { maxLength: 0 }), (value) =>
+        isEmpty(value)
+      )
+    ));
+
+  it("Should return false for non-empty arrays", () =>
+    fc.assert(
+      fc.property(fc.array(fc.nat(), { minLength: 1 }), (value) => {
+        return !isEmpty(value)
+      })
+    ))
+
+  it("Should return true for empty objects", () =>
+    fc.assert(
+      fc.property(fc.object({ maxKeys: 0 }), (value) => {
+        isEmpty(value)
+      })
+    ))
+
+  it("Should return false for non-empty objects", () =>
+    fc.assert(
+      fc.property(fc.object({ maxKeys: 200 }), (value) => {
+        !isEmpty(value)
+      })
+    ))
+
+  it("Should return true for null or undefined values", () => {
+    fc.assert(fc.property(fc.constant(null), (value) => {
```

Comment:
Pour les cas constantes tu n'es pas obligé d'utiliser fast-check, tu peux juste it("", () => assert(isEmpty(null))
Avec asset qui viens de chai je crois

Reply:
No chai no party